src/sdk/EnvoyPluginJobAttachment.ts

```
@@ -44,6 +99,7 @@ export interface EnvoyPluginCredentialJobAttachment extends EnvoyPluginTextJobAt
 type EnvoyPluginJobAttachment =
```

Do we want a non-screener EnvoyPluginJSONJobAttachment here too?

---

When theyre formatted incorrectly they just silently fail to display. Given that I don't know the accepted formats, I don't want to pretend to support some generic version that won't display.